packages/slice-machine/src/pages/changes.tsx

```diff
@@ -158,6 +186,28 @@ const Changes: React.FunctionComponent = () => {
                 void onPush(false); // not deleting documents by default
               }}
             />
+            <Toast
```

Comment:
💡 Maybe we could start using `ToastProvider` and the `useToast` hook.

Reply:
yes, I want to add that, but missed one prop when adjusting ToastProvider, so this [PR](https://github.com/prismicio/editor/pull/1193) is needed first

Reply:
actually, in will leave it like that, 
when toast is launched with useToast, it stays on page when navigating away from `/changes` page, which in general is good but not in this use-case

Reply:
yes probably, in general I have impression that Toast would need some proper revision